src/components/v5/shared/UserAvatarDetails/UserAvatarDetails.tsx

```diff
@@ -49,7 +44,12 @@ const UserAvatarDetails: FC<UserAvatarDetailsProps> = ({
               'border-purple-400': userStatus === 'top' && isContributorsList,
             })}
           >
-            <Avatar size="m" title={userName} avatar={avatar} />
+            <Avatar
+              size="m"
+              title={userName}
+              avatar={avatar}
+              mode={userStatus || 'general'}
```

Comment:
|| 'general' shouldn't be needed after changes

Reply:
unfortunately it is needed

Reply:
fixed